src/List.js

```diff
@@ -77,77 +129,79 @@ export function List() {
       setDoc(
         itemRef,
         {
           lastPurchasedDate: date.getTime(),
-          previousEstimate: calculateEstimate(
-            item.previousEstimate || parseInt(item.days),
-            daysSinceLastTransaction,
-            item.totalPurchases,
+          previousEstimate: Math.round(
+            calculateEstimate(
+              item.previousEstimate,
+              daysSinceLastTransaction,
+              item.totalPurchases,
+            ),
           ),
           totalPurchases: item.totalPurchases + 1,
         },
         { merge: true },
       );
     }
   };
-  if (items.length) {
-    return (
-      <>
-        <label htmlFor="filterItems">Filter items:</label>
-        <input
-          name="filterItems"
-          type="text"
-          value={filterItem}
-          placeholder="Start typing here..."
-          onChange={(event) => setFilterItem(event.target.value)}
-        ></input>
-        <button onClick={() => setFilterItem('')}>X</button>
-        <ul className="list">
-          {items &&
-            items
-              .filter((item) => !!item.id)
-              .filter((item) =>
-                item.name.toLowerCase().includes(filterItem.toLowerCase()),
-              )
-              .map((item) => {
-                return (
-                  <li key={item.id}>
-                    <input
-                      type="checkbox"
-                      id={`custom-checkbox-${item.id}`}
-                      name={item.name}
-                      value={item.name}
-                      checked={
-                        !!item.lastPurchasedDate &&
-                        new Date() - item.lastPurchasedDate < ONE_MINUTE
-                      }
-                      onChange={(e) => handleChange(item.id, e)}
-                    />
-                    <label htmlFor={`custom-checkbox-${item.id}`}>
-                      {item.name}
-                    </label>
-                  </li>
-                );
-              })}
-        </ul>
-        <NavigationMenu />
-      </>
-    );
-  }
-  return (
+
+  return items.length ? (
+    <>
+      <label htmlFor="filterItems">Filter items:</label>
+      <input
+        name="filterItems"
+        type="text"
+        value={filterItem}
+        placeholder="Start typing here..."
+        onChange={(event) => setFilterItem(event.target.value)}
+      ></input>
+      <button onClick={() => setFilterItem('')}>X</button>
+      <ul className="list">
+        {items &&
+          items
+            .filter((item) => !!item.id)
+            .filter((item) =>
+              item.name.toLowerCase().includes(filterItem.toLowerCase()),
+            )
+            .sort(itemSort)
+            .map((item) => {
+              return (
+                <li key={item.id} className={itemState(item)}>
+                  <input
+                    type="checkbox"
+                    aria-label={itemState(item)}
```

Comment:
I believe that we don't need an `aria-label` when we have a `<label />` that is associated with the input. If we were to use an `aria-label`, I think that we'd want it to be `item.name`

Reply:
Thanks Mike! In the criteria notes it says:

"Be sure to not only use color to denote how soon the user will need to buy the item, as noted in the wireframe. https://github.com/the-collab-lab/smart-shopping-list/blob/main/_resources/wireframes/list-populated.png "

If you go to the wireframe, it says to use the aria label to indicate how soon we need to buy it.

Is there a different way we could indicate this? item.name I think would just return the item's name, which is not what we're trying to do here.

Reply:
ahh...you're totally right. ignore me!